liquibase-core/src/main/java/liquibase/change/core/ExecuteShellCommandChange.java

```diff
@@ -154,12 +146,12 @@ public Sql[] generate(Database database) {
                         throw new UnexpectedLiquibaseException("Error executing command: " + e.getLocalizedMessage(), e);
                     }
 
-                    return null;
+                    return new Sql[0];
```

Comment:
Is this change is expected?

Reply:
I followed a suggestion made by Sonar, but I probably shouldn't have made that change.  I will revert it, just to be safe.